bigquery2/jwt_issuer.bal

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/encoding;
+import ballerina/internal;
+import ballerina/crypto;
+
+# Issue a JWT token.
+#
+# + header - JwtHeader object
+# + payload - JwtPayload object
+# + config - JWTIssuerConfig object
+#
+# + return - JWT token string or an error if token validation fails
+public function issue(internal:JwtHeader header, internal:JwtPayload payload, internal:JWTIssuerConfig config)
+                                                                                       returns (string|error) {
```

Comment:
Let's improve the formatting as well. Seems like there are too many leading white spaces.